src/routes/PrivateRoute.tsx

```diff
@@ -6,6 +6,7 @@ function PrivateRoute(props: RouteProps) {
   if (currentUser) {
     return <Route {...props} />;
   }
+  sessionStorage.setItem("boardId", props.location!.pathname);
```

Comment:
This call of `setItem` will only store the relative path or board id and might lead to issues once you land on a different path